Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -509,6 +509,8 @@
   SmmPolicyGateLib|MmSupervisorPkg/Library/SmmPolicyGateLib/SmmPolicyGateLib.inf
   HwResetSystemLib|QemuQ35Pkg/Library/ResetSystemLib/StandaloneMmResetSystemLib.inf
   IhvSmmSaveStateSupervisionLib|MmSupervisorPkg/Library/IhvMmSaveStateSupervisionLib/IhvMmSaveStateSupervisionLib.inf
+  MmServicesTableLib|StandaloneMmPkg/Library/StandaloneMmServicesTableLib/StandaloneMmServicesTableLibCore.inf
+  MmSaveStateLib|UefiCpuPkg/Library/MmSaveStateLib/AmdMmSaveStateLib.inf

```

Comment:
I haven't looked at the libraries instances, but it looks like Q35 is expected to use the AMD one?

Reply:
I think the Q35, if not specifying the CPU, is defaulting to an AMD processor.

Reply:
This reminded me of https://github.com/microsoft/mu_tiano_platforms/pull/669.

`CpuInfoDxe` is currently printing:

```
  CPU Brand Name: QEMU Virtual CPU version 2.5+
  Family Id: 0x7CACF080
  Stepping: 0x1
  Model Id: 0x60FB0
```

That is the default model: https://www.qemu.org/docs/master/system/i386/cpu.html#default-x86-cpu-models

I guess that is fine for our default. Though it seems like this could fail if a different (Intel) CPU is used.

Reply:
Discussed this morning offline. There is always going to be a gap between the build and the QEMU launching command, as long as the library is divided into AMD vs. Intel processors. For now, we will stick with the AMD processor, because this is the default option when launching our Q35 platform image.